changes/api/502.compose-and-kana.feature.md

```diff
@@ -0,0 +1 @@
+Add `XKB_LED_NAME_COMPOSE` and `XKB_LED_NAME_KANA` definitions to conform to USB HID.
```

Comment:
```suggestion
`xkbcommon-names.h`: Add `XKB_LED_NAME_COMPOSE` and `XKB_LED_NAME_KANA` definitions
to cover all LEDs defined in [USB HID](https://usb.org/sites/default/files/hid1_11.pdf).

Contributed by Martin Rys
```

The *conformance* to HID is at driver level, not here. `libxkbcommon` just provides theses names as a convenience, but the actual indexes are defined in `xkeyboard-config`, as commented previously.

Reply:
Also please change the file name to `+compose-and-kana-led-names.features.md`, as `towncrier` cannot handle both issue and PR ID and we chose issue ID.